javascript-modal/main.js

```diff
@@ -0,0 +1,11 @@
+const openModal = document.querySelector('.open-modal');
+const closeModal = document.querySelector('.close-modal');
+const modalContainer = document.querySelector('.modal-container');
+
+openModal.addEventListener('click', function () {
+  modalContainer.style.display = 'flex';
+});
+
+closeModal.addEventListener('click', function () {
+  modalContainer.style.display = 'none';
+});
```

Comment:
It's fine for this exercise, but we want to be careful and avoid inline styles as much as possible, because it will override any css that gets applied in our stylesheets.